bot/tasks/pennychat.py

```diff
@@ -197,11 +197,13 @@ def _penny_chat_details_blocks(penny_chat_invitation, mode=None):
     if include_calendar_link:
         start_date = penny_chat_invitation.date.astimezone(utc).strftime('%Y%m%dT%H%M%SZ')
         end_date = (penny_chat_invitation.date.astimezone(utc) + timedelta(hours=1)).strftime('%Y%m%dT%H%M%SZ')
+
+        description = f'{penny_chat_invitation.description} [Video Link]({penny_chat_invitation.video_conference_link})'
```

Comment:
maybe add a new line? `...description}\n[Video...`

Reply:
Lol, I totally forgot to revisit this. Adding to todos...